src/custom-element-widget.js

```diff
@@ -159,6 +176,27 @@ function makeUpdate() {
   };
 }
 
+function makeDestroy() {
+  return function destroyCustomElement(element) {
+    // Dispose propertiesProxy
+    let proxiedProps = element.cycleCustomElementMetadata.propertiesProxy;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).